servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -359,6 +394,13 @@ private FileInfo getFileInfoInternal(Inode inode) throws FileDoesNotExistExcepti
   }
 
   /**
+   * @return a read-only view of the inode tree
```

Comment:
what is the relation ship between inode tree and file store? given the method-level comment, should the method be called getInodeTreeView?